benefit-finder/package.json

```diff
@@ -34,7 +34,11 @@
     "cy:run": "NODE_ENV=test npm run cy:run:component && npm run cy:run:e2e",
     "cy:run:prod:e2e": "NODE_ENV=test npx cypress run --config-file cypress.prod.config.js --browser chrome",
     "cy:run:prod:links:e2e": "NODE_ENV=test npx cypress run --config-file cypress.prod.links.config.js --browser chrome",
-    "cy:run:pipeline": "NODE_ENV=test concurrently -k -s first -n \"SB,TEST\" -c \"magenta,blue\"   \"http-server ./storybook-static --port 6006 --silent\"   \"wait-on tcp:6006 && npm run cy:run\" -Y"
+    "cy:run:pipeline": "NODE_ENV=test concurrently -k -s first -n \"SB,TEST\" -c \"magenta,blue\"   \"http-server ./storybook-static --port 6006 --silent\"   \"wait-on tcp:6006 && npm run cy:run\" -Y",
+    "cy:run:edge:e2e": "NODE_ENV=test npx cypress run --browser edge",
```

Comment:
why do we pass these browser flags instead of just configuring in the pipeline with the action?

https://glebbahmutov.com/blog/cypress-webkit/